lib/src/components/dropdown/dropdown.dart

```diff
@@ -0,0 +1,440 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+import '../../../zeta_flutter.dart';
+
+/// Class for [ZetaDropDown]
```

Comment:
@thelukewalton where should we be getting component descriptions from? There's nothing on the figma for dropdown but also we should have something here

Reply:
I'll message Greg about this - I have been using chat gpt to make up some vaguely relevant text, but we should phase that out